aggregator/src/main/java/com/redhat/cloud/notifications/db/EmailAggregationResources.java

```diff
@@ -15,29 +17,55 @@
 public class EmailAggregationResources {
 
     @Inject
-    Session session;
+    EntityManager entityManager;
+
+    private static final String DEFAULT_ORG_ID = "NA";
+
+    public List<EmailAggregationKey> getApplicationsWithPendingAggregation(String orgIdToAggregate, LocalDateTime start, LocalDateTime end) {
+        String orgIdCriteria = "ea.org_id = :orgId AND acj.org_id IS NOT NULL";
+        if (DEFAULT_ORG_ID.equals(orgIdToAggregate)) {
+            orgIdCriteria = "acj.org_id IS NULL";
+        }
+        String query = "SELECT DISTINCT ea.org_id, ea.bundle, ea.application FROM email_aggregation ea "
+                + "LEFT JOIN aggregation_cronjob_parameter acj ON ea.org_id = acj.org_id "
+                + " WHERE " + orgIdCriteria + " AND ea.created > :start AND ea.created <= :end";
```

Comment:
Wouldn't this work?
```suggestion
                + " WHERE ea.org_id = :orgId AND ea.created > :start AND ea.created <= :end";
```
and then below
```
.setParameter("orgId", orgIdToAggregate)
```

Reply:
Please ignore that comment for now.